ServerCore/wwwroot/css/pd2022/base-styles.css

```diff
@@ -23,7 +23,7 @@
 
 @font-face {
   font-family: "pd2022-font-title";
-  src: url("resources/LTtilt.ttf");
+  src: url("https://puzzleserverteststore.blob.core.windows.net/evt4/fHdElKHm5bEGdqMFPRvZjA==/LTtilt.ttf");
```

Comment:
evt4??? which is that - unsure what is happening here and how it keeps from breaking stuff. If this is the pdalpha event, this means that everything will break when we clean it for PD2023.